node/src/Pos/Client/Txp/Util.hs

```diff
@@ -473,9 +476,10 @@ computeTxFee
     -> TxOutputs
     -> TxCreator m TxFee
 computeTxFee utxo outputs = withLinearFeePolicy $ \linearPolicy -> do
-    txAux <- createFakeTxFromRawTx <$>
-             stabilizeTxFee linearPolicy utxo outputs
-    txToLinearFee linearPolicy txAux
+    TxRaw {..} <- stabilizeTxFee linearPolicy utxo outputs
+    let outAmount = sumTxOutCoins trOutputs
+        inAmount = sumCoins $ map (txOutValue . fst) trInputs
+    integerToFee $ inAmount - outAmount
```

Comment:
We already have ready fees value in `stabilizeTxFee`, just don't return it. Why not to use it?